app/models/pacbio/well.rb

```diff
@@ -9,13 +9,8 @@ class Well < ApplicationRecord
     include Uuidable
     include SampleSheet::Well
 
-    # We should get rid of the below enum,
-    # as we are not supporting sequencing_mode in SMRTLink v10
-    # but if we keep the field in the database, for auditing,
-    # then we need this to translate the integer to a value
-    enum sequencing_mode: { 'CLR' => 0, 'CCS' => 1 }
-
-    enum generate_hifi: { 'In SMRT Link' => 0, 'On Instrument' => 1, 'Do Not Generate' => 2 }
+    # Do not delete until the column has been migrated
```

Comment:
Is there a ticket that can be referenced here for that migration?  I can see this staying here for the long term without something referencing when it is removable.

Reply:
I can remove it as it is referenced in the migration